packages/bitcore-node/src/models/webhook.ts

```diff
@@ -0,0 +1,54 @@
+import { ObjectID } from 'mongodb';
+import { StorageService } from '../services/storage';
+import { BaseModel } from './base';
+
+export interface IWebhook {
+  _id?: ObjectID;
+  chain: string;
+  network: string;
+  source: string;
+  sourceId: string;
+  tag?: string;
+  body: any;
+  timestamp: Date;
+  processed: boolean;
+}
+
+export class WebhookModel extends BaseModel<IWebhook> {
+  constructor(storage?: StorageService) {
+    super('webhook', storage);
+  }
+  allowedPaging = [];
+
+  onConnect() {
+    // capped at 0.5 million docs
+    this.db?.createCollection(this.collectionName, { capped: true, size: 500000 })
```

Comment:
Note: MongoDB interprets this as a size in bytes, not a document count. So this is 50 mb

Reply:
Nice catch! Actually that should be 500 kB. I increased to 100 MiB